.travis.yml

```diff
@@ -14,32 +14,78 @@ go_import_path: github.com/kubeless/kubeless
+    - secure: qbbTVfwkOraWTkQ8SxJOAsjv7qVrWqrOINIOCeFcKzwrQPHVjzr97JeOcQK73PLTwkgYg8QurwiRGB98O0fcroiqbzgXIwVk0yFrRtbmcaQ/NYXt9ipfPeeChsn8bwF7vYRuGfz9HybCU2CBmxwjJqfTeU2iqvOFfAH2Sh4b6VmbCVkINn4GFHLATcwHKoQO16B6S+z4FGSHX/AWT3vRFTBZZ65gZF7evPURtNDbumwe28KCKzmqySF5UKw2HEmDyCD+fRozlCcWYC2lbS2iZn/d0VXZvUpZxpEDVyaHdH/n6I03DZ+gnbgCi+/uw7d96s3pjomPkCiY9VfmQPl020kR8FeREyOsc0rqYtrxwe1ImKJUHsjdUZ6nUL5HcFyWJdKX6txn6bUJwUOF/+lg1O34n0ZJym3j5sm7kyoBzh0lTRMiue7Nd9WD7iJePmPEfb+Sk6UGCjgf/KGkcGr75jaVFb3AsIjtGgbq9BDQN4SKRGW3NSmXZ6pKbwFm1b6c+LPPOTukKsPTUBwEqcCCwoNSqcgIBv325dBEKJMa945/RR7E7/KXzKoWkoLkKZUTPKtVJpjsIxVkq1spPZQmmSZ3FBUww2tC+l6stD62xDYSuBH6iqLhU/xpeAjizCMS1YjNUxjs1oVBkdu2IEUV61SOoZtYyIgDvvPGpm2YcgQ=
+    - secure: HbpGKPXLPT2t24YUXuk/A4Sb0K5z1AdsTz9Xo3loQr/yuR07kDhkhV5qkrgDO7wL+4VD+n1vEwtTIqnSAwWmcP0fxaWO56efgAhstXz117wcHAcfJgFTPuwMsZKqWNPc5T4yeyUNlx8UZcs1a6oCHPH2Q8fF5dOukJ74JS82YU1KIxa+TpziFaBZ5Ti/WHkCtaNxd15u3vneRUnXmBY9XUDLelSTwmUXFUYwtDYezF9U9yr8LlZoRVlkNe1NT1YQoWLmi2UaEo4rVjOisKlL8/ejOeciL6o8zYJ2XdKO58AvdyDJi90/txYjHhPJiAcL996EWIBNkg0Ms2Y6QgIpitOD8/HrxB0qONFX4suv5XDG/u3fO++JA1FTXFlQzL8/ulG94YcCyivDBQfmUb3KVNOpU1JZKqJY7bfGZwg90IJSirjixpGMh99Gm9R2WNLjPvIkjBZug3ZGiVgVL3PR4UGGkeKYYzNX+SWcTdC2ECu8HtbF/8HklpWB69gf+lN9xGgZE31+Yi3sd/zcIeECiOW3AaIGVoYOSuf+iXooG44BqFS9kHQtx2oKu73uS9/5jFu5TqU5B3v7Yryb3VHRrKqwkRAJZDl/Sdq9pbycS0xAzZErrU52iiAj5SNJyhOI+SojeuWuxo4O40w1wKi742VEWqKPHcYKYixNz+RvJEY=
+    - secure: CXyDSLR6KJGoPN4ApD4UjIqXxKWxvbSuun1Py3ViI6XuhJOjTBlNJVvCV3cZUD+Dvs98sUnNVvrZgoP/YxCIF6tPzm6VboiO75gxXKJQCDbOuUE9PhBiU2WhHCEXS0AL7emCgKqOw4hNWUdyzhx+lytZkIrFFwuT0qUVqCF/1ccDJXEl3YeMotoMqiV+XQZIvoMNQVY8ENJvYNoBWi+iFF+lSRBUMavB/4soXvZjASMfz189/BQTGfXeOeKu6E+O51zup7Z+W28b9bO1r47jjY8VFdLrnnfDPtN7qEd/An4OkICnijcMHbS4n1NO9mb/wzYvOuo7hDWU5cHDbN/rhaoFmHa/8Qp9N8UMza70ps4mO4jYg+e9ZTGZvFMRU3gcF7XRkrdqBFQOsgTVJrZsfzRFiV5M30PXv4gc7UXVvsHrSj76bRpQL+dkGdqIQZxc9OFAOJ/dRLwGowu1dS5N63444xGx5PuC5IZ81Z4qIDfzpAfM5637NHrkNT9KRFyJktAK8XK1e7c1wD95uFyEDv9UPziJLEPOtTP8/YLSs1H0TgjVAyStv6HcqHDHff5n25rFehNlULpJcktqKv1YEyYGak9nwGsSTk2h73kvKp2cAfLdlW9vN9IxF60G+G+OmB4ycK+KfU0FNg/NVm40+LeogIWgz9Xh3LV+obEeMC0=
 before_install:
   - set -e
 
 install:
   - make bootstrap
-
-script:
-  - make test
-  - make validation
-  - make VERSION=${TRAVIS_TAG:-build-$TRAVIS_BUILD_ID} binary
   - |
-    if [ "$TRAVIS_OS_NAME" = linux ]; then
-      make controller-image CONTROLLER_IMAGE=$CONTROLLER_IMAGE
+    if [[ "$TEST_TARGET" == "GKE" && -n "$GKE_ADMIN" && "$TRAVIS_EVENT_TYPE" != "pull_request" ]]; then
+      export GOOGLE_APPLICATION_CREDENTIALS=$TRAVIS_BUILD_DIR/client_secrets.json
+      openssl aes-256-cbc -K $encrypted_1dbafa65b3ad_key -iv $encrypted_1dbafa65b3ad_iv -in .github/gcloud.json.enc -out $GOOGLE_APPLICATION_CREDENTIALS -d
+      if [ ! -d $HOME/gcloud/google-cloud-sdk ]; then
+        mkdir -p $HOME/gcloud &&
```

Comment:
I will try 👍

Reply:
I tried but it failed because of the input required: https://travis-ci.org/kubeless/kubeless/jobs/318127686#L530
I will go back to this verbose solution

Reply:
huh, strange that it worked for their example, unless it's out of date. Ah well, this is fine